db/migrate/20221111224025_add_mappings_to_schema.rb

```diff
@@ -0,0 +1,5 @@
+class AddMappingsToSchema < ActiveRecord::Migration[5.2]
+  def change
+    add_column :allinson_flex_profile_properties, :mappings, :string unless column_exists?(:allinson_flex_profile_properties, :mappings)
```

Comment:
Since this will be a hash, I'm curious if a string is the correct data type here. 🤔

Reply:
Supported types are listed below. I don't know what other type would be more appropriate? You can see how string makes the data look in the screenshot above. 

- :binary
- :boolean
- :date
- :datetime
- :decimal
- :float
- :integer
- :primary_key
- :string
- :text
- :time
- :timestamp

Reply:
Ah never mind, I see that your link specified using text. I can switch to that.